junit5/src/main/java/org/jboss/weld/junit5/WeldJunit5Extension.java

```diff
@@ -162,6 +171,10 @@ public Object resolveParameter(ParameterContext parameterContext, ExtensionConte
     @Override
     public boolean supportsParameter(ParameterContext parameterContext, ExtensionContext extensionContext)
             throws ParameterResolutionException {
+        // don't resolve parameters again that JUnit already resolves
```

Comment:
The `again` is kind of misleading, I'd reword as follows:
```suggestion
        // Do not attempt to resolve JUnit 5 built-in parameters
```

Reply:
good catch, agree, changed